### ISSUE TITLE
Node version sensitivity

### ISSUE THREAD

**Issue by jmgasper** [topcoder-platform/topcoder-healthcheck-dropin#19]
I was changing the challenge processor ES to use the new Opensearch stuff.  I had to update Node to be able to use the opensearch NPM library, so I upgraded the Dockerfile to node:18.14.1-alpine3.17
I deployed the changes and everything looked good, but when QA got to testing it, it didn't work for them.  I investigated and saw that the ECS cluster had reverted the task definition back to a previous "known good" version, which was odd because I tested it as working.  After digging into it, I found out that the health check had stopped working, so ECS couldn't determine that the new version was running properly (even though it was), and it was rolling back.  I updated the health check dependency version in the package.json, and that didn't fix it, so I went and compared with other processors and changed the node version to node:18.16.1, and it started working again.
So, long story short - the health check dropin appears to be mildly sensitive to node versions, causing subtle rollback issues. (edited)
